Cargo.toml

```diff
@@ -72,3 +72,7 @@ itertools = "0.10"
 serde = { version = "1.0.139" }
 serde_json = "1.0.64"
 tracing = "0.1"
+
+[patch.crates-io]
```

Comment:
I'm not thrilled about this patch section, but we need it to be able to import types from `renegade-contracts::common`. I'm hoping to be able to remove the need for this when we unify types later, but if it's concerning, I can try thinking about this now

Reply:
Why not just mock them out? This has a pretty large impact on the relayer as a whole and is not worth what is a relatively minor benefit in "having the types be the same".

Reply:
it's more about not having to redefine serialization for the types than just being able to import them from elsewhere. the serialization derivation in `renegade-contracts` is quite hairy, redefining it here increases the surface area of potential inconsistencies... but I agree that I don't want this patch to stick around, so I'll redefine serde stuff + the types in the relayer for now, and this may be a good base / source of truth for the unification later